Gps crash on latest alpha

```
type: crash
osVersion: google/husky/husky:16/BP2A.250705.008/2025070800:user/release-keys
package: com.google.android.gms:252635035, targetSdk 36
process: com.google.android.gms.ui
processUptime: 225 + 345 ms
installer: app.grapheneos.apps
GmsCompatConfig version: 158

java.lang.RuntimeException: Unable to start activity ComponentInfo{com.google.android.gms/com.google.android.gms.advancedprotection.ui.AdvancedProtectionSettingsActivity}: java.lang.SecurityException: Access denied, requires: android.permission.MANAGE_ADVANCED_PROTECTION_MODE
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4293)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:4480)
	at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:222)
	at android.app.servertransaction.TransactionExecutor.executeNonLifecycleItem(TransactionExecutor.java:133)
	at android.app.servertransaction.TransactionExecutor.executeTransactionItems(TransactionExecutor.java:103)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:80)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2832)
	at android.os.Handler.dispatchMessage(Handler.java:110)
	at android.os.Looper.loopOnce(Looper.java:248)
	at android.os.Looper.loop(Looper.java:338)
	at android.app.ActivityThread.main(ActivityThread.java:9106)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:593)
	at com.android.internal.os.ExecInit.main(ExecInit.java:50)
	at com.android.internal.os.RuntimeInit.nativeFinishInit(Native Method)
	at com.android.internal.os.RuntimeInit.main(RuntimeInit.java:371)
Caused by: java.lang.SecurityException: Access denied, requires: android.permission.MANAGE_ADVANCED_PROTECTION_MODE
	at android.os.Parcel.createExceptionOrNull(Parcel.java:3349)
	at android.os.Parcel.createException(Parcel.java:3333)
	at android.os.Parcel.readException(Parcel.java:3309)
	at android.os.Parcel.readException(Parcel.java:3251)
	at android.security.advancedprotection.IAdvancedProtectionService$Stub$Proxy.getAdvancedProtectionFeatures(IAdvancedProtectionService.java:281)
	at android.security.advancedprotection.AdvancedProtectionManager.getAdvancedProtectionFeatures(AdvancedProtectionManager.java:336)
	at aafh.D(:com.google.android.gms@252635035@25.26.35 (260400-783060121):192)
	at nnf.onCreate(:com.google.android.gms@252635035@25.26.35 (260400-783060121):74)
	at fy.e(:com.google.android.gms@252635035@25.26.35 (260400-783060121):722)
	at fp.aL(:com.google.android.gms@252635035@25.26.35 (260400-783060121):1139)
	at fp.aN(:com.google.android.gms@252635035@25.26.35 (260400-783060121):83)
	at fp.O(:com.google.android.gms@252635035@25.26.35 (260400-783060121):98)
	at cd.f(:com.google.android.gms@252635035@25.26.35 (260400-783060121):7)
	at com.google.android.gms.advancedprotection.ui.AdvancedProtectionSettingsChimeraActivity.onCreate(:com.google.android.gms@252635035@25.26.35 (260400-783060121):26)
	at wfs.public_onCreate(:com.google.android.gms@252635035@25.26.35 (260400-783060121):1)
	at augf.onCreate(:com.google.android.gms@252635035@25.26.35 (260400-783060121):4)
	at android.app.Activity.performCreate(Activity.java:9208)
	at android.app.Activity.performCreate(Activity.java:9186)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1531)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:4275)
	... 15 more
```

--- a/gmscompat_config
+++ b/gmscompat_config
@@ -206,6 +206,11 @@ getUserRestrictionSources emptyList
 [android.safetycenter.SafetyCenterManager]
 isSafetyCenterEnabled false
 
+[android.security.advancedprotection.AdvancedProtectionManager]
+getAdvancedProtectionFeatures emptyList
+setAdvancedProtectionEnabled void
+logDialogShown void
+
 [android.security.keystore.recovery.RecoveryController]
 initRecoveryService void
 

